src/v1/services/tenantService.ts

```diff
@@ -0,0 +1,14 @@
+import { Tenant } from '../models/tenant';
+import { AppDataSource } from '../config';
+import { Optional } from 'sequelize';
+
```

Comment:
please add code comments

Reply:
updated with comment

Reply:
this is for all the code files. added this comment for reference. add comments for all the logics written

Reply:
spell check